src/plugins/saved_objects_management/tsconfig.json

```diff
@@ -13,13 +13,11 @@
   ],
   "references": [
     { "path": "../../core/tsconfig.json" },
-    { "path": "../dashboard/tsconfig.json" },
     { "path": "../data/tsconfig.json" },
-    { "path": "../discover/tsconfig.json" },
     { "path": "../home/tsconfig.json" },
     { "path": "../kibana_react/tsconfig.json" },
     { "path": "../management/tsconfig.json" },
-    { "path": "../visualizations/tsconfig.json" },
+    { "path": "../saved_objects_tagging_oss/tsconfig.json" },
```

Comment:
We were indirectly inheriting the `saved_objects_tagging_oss` types from the `visualizations` plugin. With the `visualizations` dependency gone, we need to declare it explicitly.